src/main/java/com/ericsson/ei/jmespath/JmesPathInterface.java

```diff
@@ -67,9 +76,9 @@ public JsonNode runRuleOnEvent(String rule, String event) {
             result = expression.search(eventJson);
         } catch (Exception e) {
             String msg = "runRuleOnEvent failed for given arguments:\n";
-            msg += "rule was: " + rule + "\n";
-            msg += "event was: " + event + "\n";
-            log.error(msg, e);
+            msg += "Rule was: " + rule + "\n";
+            msg += "Event was: " + event + "\n";
+            LOGGER.error(msg, e);
```

Comment:
Here I would want you to build the string using for example String.format() or use the Logger built in formatter, instead of building a string with "+" and addition.

Reply:
I'll fix it